Documentation/tutorial/docfx_getting_started.md

```diff
@@ -64,7 +64,27 @@ namespace WebApplication1
 
 *Step4.* Right click on the website project, and click *View* -> *View in Browser*, navigate to `/_site` sub URL to view your website!
 
-4. Build from source code
+4. Use *DocFX* with a Build Server
+---------------
+
+*DocFX* can be used in a Continuous Integration environment. 
+
+Most build systems do not checkout the branch that is being built, but 
+use a `detached head` for the specific commit.  DoxFX needs the the branch name to implement the `View Source` link in the API documentation.
+
+Setting the environment variable `DOCFX_SOURCE_BRANCH_NAME` tells DocFX which branch name to use.
+
+Many build systems set an environment variable with the branch name.  DocFX uses the following:
```

Comment:
Support `DOCFX_SOURCE_BRANCH_NAME` is enough? Use documentation to tell CI users how to configure environment variables with different build systems?